.travis.yml

```diff
@@ -1,14 +1,18 @@
-# GCE VMs have better performance (will be upgrading to premium VMs soon)
-sudo: required
-
+dist: xenial
+group: stable
```

Comment:
so did we decide to keep those two entries in the end?

Reply:
I did attempt to removed it, but for some reason Travis CI picked up trusty - https://travis-ci.org/scala/scala/jobs/556437704#L7-L9

Reply:
OK. Thanks!